src/hooks/useMarkdownStyle.ts

```diff
@@ -26,11 +26,13 @@ function useMarkdownStyle(): MarkdownStyle {
             },
             code: {
                 fontFamily: FontUtils.fontFamily.platform.MONOSPACE,
+                fontSize: 13, // TODO: should be 15 if inside h1, see StyleUtils.getCodeFontSize
```

Comment:
Currently, `react-native-live-markdown` supports only one fixed value for font size for each individual Markdown syntax style. However, the actual font size depends on whether the inline code is inside h1 or not:

https://github.com/Expensify/App/blob/7177173f4b686a32bfcb94f07f0541b2f865f31b/src/styles/utils/index.ts#L595-L600

I suggest that we use 13 for now as we need more time to implement it properly on Live Markdown side.

Here's how it looks with 13:

<img width="222" alt="Screenshot 2024-04-03 at 17 37 24" src="https://github.com/Expensify/App/assets/20516055/cec799a3-8f80-47ed-aae1-7b451af4f1a2">

Reply:
@thienlnam What do you think? Can we skip this edge case as for now?

Reply:
Yeah this seems fine to skip